pkg/logflags/logflags.go

```diff
@@ -177,8 +180,11 @@ func Setup(logFlag bool, logstr string, logDest string) error {
 			fnCall = true
 		case "minidump":
 			minidump = true
-			// If adding another value, do make sure to
-			// update "Help about logging flags" in commands.go.
+		default:
+			if !unKnownHasPrint { // Just print first unknown component.
```

Comment:
I don't think we need to have this restriction, I don't mind printing a warning for each unknown log item.

Reply:
I have try, it looks ok. I will change and rebase.